test/e2e/rollout_test.go

```diff
@@ -542,6 +547,133 @@ var _ = Describe("placing wrapped resources using a CRP", Ordered, func() {
+			crp.Spec.Policy = &placementv1beta1.PlacementPolicy{
+				PlacementType: placementv1beta1.PickFixedPlacementType,
+				ClusterNames: []string{
+					memberCluster1EastProdName,
+				},
+			}
+			crp.Spec.Strategy.RollingUpdate.UnavailablePeriodSeconds = ptr.To(60)
+			Expect(hubClient.Create(ctx, crp)).To(Succeed(), "Failed to create CRP")
+		})
+
+		It("should update CRP status as expected", func() {
+			crpStatusUpdatedActual := customizedCRPStatusUpdatedActual(crpName, wantSelectedResources, []string{memberCluster1EastProdName}, nil, "1", false)
+			Eventually(crpStatusUpdatedActual, longEventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update CRP status as expected")
+		})
+
+		It("binding should be created", func() {
+			// List all bindings associated with the given CRP.
+			bindingList := &placementv1beta1.ClusterResourceBindingList{}
+			labelSelector := labels.SelectorFromSet(labels.Set{placementv1beta1.CRPTrackingLabel: crpName})
+			listOptions := &client.ListOptions{LabelSelector: labelSelector}
+			err := hubClient.List(ctx, bindingList, listOptions)
+			Expect(err).To(BeNil(), "Failed to list bindings")
+			Expect(len(bindingList.Items)).Should(Equal(1), "Expected one binding to be created")
+			for i := condition.OverriddenCondition; i < condition.TotalCondition; i++ {
+				cond := bindingList.Items[0].GetCondition(string(i.ResourceBindingConditionType()))
+				Expect(cond).ShouldNot(BeNil(), "Expected condition to be set", i.ResourceBindingConditionType())
+				previousAvailableTransitionTime[string(i.ResourceBindingConditionType())] = cond.LastTransitionTime
+			}
+		})
+
+		It("should place the resources on member clusters", func() {
+			workResourcesPlacedActual := waitForTestResourceToBePlaced(memberCluster1EastProd, &testCustomResource)
+			Eventually(workResourcesPlacedActual, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to place work resources on member cluster %s", memberCluster1EastProd.ClusterName)
+		})
+
+		It("should update custom resource", func() {
+			Eventually(func() error {
+				var cr testv1alpha1.TestResource
+				err := hubClient.Get(ctx, types.NamespacedName{Name: testCustomResource.Name, Namespace: workNamespace.Name}, &cr)
+				if err != nil {
+					return err
+				}
+				cr.Spec.Foo = "bar"
+				return hubClient.Update(ctx, &cr)
+			}, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update custom resource")
+		})
+
+		It("should update CRP status as expected", func() {
+			crpStatusUpdatedActual := customizedCRPStatusUpdatedActual(crpName, wantSelectedResources, []string{memberCluster1EastProdName}, nil, "2", false)
+			Eventually(crpStatusUpdatedActual, longEventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update CRP status as expected")
+		})
+
+		It("should update binding transition time", func() {
+			// List all bindings associated with the given CRP.
```

Comment:
e2e should test UX instead of internal implementation details. It should see the resources are not placed before the "notReady" time instead